libs/oidc/admin/ngx/src/lib/modules/roles/edit-role/edit-role.component.ts

```diff
@@ -23,7 +24,7 @@ export class EditRoleComponent implements OnInit {
   public deleteRole(role: Role) {
     console.log('deleteRole', role);
     this.rolesService.deleteRole(role).subscribe(() => {
-      console.log('Deleted ', role.guid);
+      this.notificationService.preset('deleted_role');
       this.fetchRoles();
```

Comment:
The pattern of `this.fetchRoles()` to assigned a new observable value is discouraged. Should not be overriding the value of the variable manually but instead next-ing the value OR what should be done in this case, setup some kind of refresh signal (subject) that fetches roles on rule service update when next-ed to.

Changes in this file are simple and would probably make you wonder "why if it works?". In this case, you'd save one line of code, but the moment you start adding additional operations or mutations to the data, you'll find yourself in a rats next if your manually calling the fetch roles, and then doing something with that data before assigning it.

Reply:
Done.